docs/generated/CLIOptions.rst

```diff
@@ -44,6 +44,8 @@
 
 --coverage-info string		Path to the coverage info file (LLVM's profdata)
 
+--include-not-covered		Include (but do not run) not covered mutants. Disabled by default
```

Comment:
nit: should this not say "non-covered" mutants? "not covered" sounds a little strange in plural.

Reply:
Yeah, it makes sense. There are some contexts (like "Include (but do not run) not covered mutants." above) where "not" is more appropriate, imo. So I think it's better to stick to one form to avoid any confusion.